package.json

```diff
@@ -60,8 +60,10 @@
     "build:esm": "esbuild --platform=node --bundle --target=es2022 --format=esm --outfile=dist/express-slow-down.mjs source/express-slow-down.ts",
     "build:types": "dts-bundle-generator --out-file=dist/express-slow-down.d.ts source/express-slow-down.ts && cp dist/express-slow-down.d.ts dist/express-slow-down.d.cts && cp dist/express-slow-down.d.ts dist/express-slow-down.d.mts",
     "compile": "run-s clean build:*",
-    "lint": "xo",
-    "format": "xo --fix",
+    "lint:code": "xo",
+    "lint:rest": "prettier --check .",
+    "format:code": "xo --fix",
+    "format:rest": "prettier --write .",
```

Comment:
I had it set up like this initially, and it seemed to cause some thrashing in the .ts files, so I think it's really formatting everything, and not using the exact same rules as xo.

Reply:
I think the problem was the mismatch between the version of XO, Typescript and Prettier. Should be fixed with d451e6c.

Reply:
Aah, that makes sense.